internal/system/agent/services.go

```diff
@@ -41,17 +41,17 @@ func InvokeOperation(action string, services []string) bool {
 
 		switch action {
 		case START:
-			executor.StartDockerContainerCompose(service, Configuration.ComposeUrl)
+			executor.ExecuteDockerCommands(service, "start")
 			break
 
 		case STOP:
-			ec.StopService(service)
+			executor.ExecuteDockerCommands(service, "stop")
```

Comment:
Note that the interface method `StopService` is no longer being used here, meaning it currently isn't used _anywhere_ in the application. I'm assuming this will be rectified or else the `ExecutorClient` interface will be removed.

Reply:
This also breaks the snap, as this interface is how the SMA is interacting with the snap.